episodes/2-identify-the-problem.Rmd

```diff
@@ -1,37 +1,30 @@
-
 ---
 title: "Identify the problem and make a plan"
-teaching: 0
-exercises: 0
+teaching: 50
+exercises: 20
 ---
 
-:::::::::::::::::::::::::::::::::::::: questions 
-
-- What do I do when I encounter an error?
-- What do I do when my code outputs something I don’t expect?
-- Why do errors and warnings appear in R? 
-- Which areas of code are responsible for errors? 
-- How can I fix my code? What other options exist if I can't fix it? 
-
-
-::::::::::::::::::::::::::::::::::::::::::::::::
-
-::::::::::::::::::::::::::::::::::::: objectives
+::: questions
+-   What do I do when I encounter an error?
+-   What do I do when my code outputs something I don’t expect?
+-   Why do errors and warnings appear in R?
+-   Which areas of code are responsible for errors?
```

Comment:
I'm confused by this one--makes it sound like there are certain areas that are responsible for errors in general. Was this the intent?

Reply:
Changed to "How can I find which areas of code are responsible for errors?"